src/main/java/com/glodblock/github/api/ISide.java

```diff
@@ -0,0 +1,8 @@
+package com.glodblock.github.api;
+
+import net.minecraftforge.common.util.ForgeDirection;
+
+public interface ISide {
+
+    ForgeDirection getSide();
```

Comment:
I guess AE2 is the right place to add this interface, like `AEBasePart` has a method with the same name

Reply:
I plan to do some other thing related to terminals. I think I can move this interface after

Reply:
You mean you're going to move this interface later but not in this PR?

Reply:
yes exactly